delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/AclsRoutes.scala

```diff
@@ -110,20 +110,20 @@ class AclsRoutes(identities: Identities, acls: Acls)(implicit
                   (put & entity(as[ReplaceAcl])) { case ReplaceAcl(AclValues(values)) =>
                     authorizeFor(address, aclsPermissions.write).apply {
                       val status = if (rev == 0L) Created else OK
-                      emit(status, acls.replace(Acl(address, values: _*), rev).map(_.void))
+                      emit(status, acls.replace(Acl(address, values: _*), rev).map(_.map(_.metadata)))
```

Comment:
This is quite annoying. We could have the following in some syntax:

```scala
trait IOResourceFSyntax {
  implicit final def resourceFSyntax[E, A](io:IO[E, ResourceF[A]]): IOResourceFOps = new IOResourceFOps(io)
}
final class IOResourceFOps[E, A](private val io:IO[E, ResourceF[A]]) extends AnyVal {
   def mapValue[B](f: A => B): IO[E, B] = io.map(_.map(f))
}
```

And then use `mapValue`

Something like that

Reply:
Or maybe a more generic one, with Functor:
```scala
trait IOFunctorSyntax {
  implicit final def ioFunctorOps[E, A: Functor](io:IO[E, A]): IOFunctorOps = new IOFunctorOps(io)
}
final class IOFunctorOps[E, A: Functor](private val io:IO[E, A]) {
   def mapValue[B](f: A => B): IO[E, B] = io.map(_.map(f))
}
```